core/cmd/client.go

```diff
@@ -92,30 +91,16 @@ func (cli *Client) errorOut(err error) error {
 	return nil
 }
 
-func (cli *Client) setConfigFromFlags(opts *chainlink.GeneralConfigOpts, ctx *cli.Context) error {
-
-	configToProcess := ctx.IsSet("config") || ctx.IsSet("secrets")
-	if !configToProcess {
-		return nil
```

Comment:
When no flags were used, this would short circuit both times, so that `loadOpts` never runs to process the `CL_CONFIG` env var.